ndb_api/db_helpers.go

```diff
@@ -548,6 +464,43 @@ func (a *PostgresHARequestAppender) appendProvisioningRequest(req *DatabaseProvi
 	return req, nil
 }
 
+func defaultActionArgumentsforHAProvisioning(database DatabaseInterface, dbPassword string, databaseNames string) map[string]string {
```

Comment:
Maybe this function should be called setActionArgumentsforHAProvisioning? since its only setting defaults if values not given?